packages/ux-capture/examples/micro_spa.js

```diff
@@ -25,26 +25,27 @@ setTimeout(() => {
 			'<i>... loading page 2 ...</i>';
 
 		setTimeout(() => {
-			[
-				'ux-text-intro',
-				'ux-text-story',
-				'ux-image-inline-kitten',
-				'ux-image-onload-kitten',
-				'ux-text-story-details-link',
-				'ux-handler-moreclickable',
-				'ux-text-page2',
-				'ux-handler-page2-clickable',
-			].map(UXCapture.clearMarks);
```

Comment:
do we no longer need to call `clearMarks`?

Reply:
Yes, the idea is that we are not going to check if mark pre-exists to determine if element is already on the page, but will use selectors to find it in DOM instead.

This has two benefits:
- it will keep the history of marks intact
- will require less coding on engineer's part (it was easy to clear marks in React's lifecycle methods, but much harder in other environments)